ILSpy/TreeNodes/AssemblyTreeNode.cs

```diff
@@ -687,4 +687,35 @@ public void Execute(TextViewContext context)
 		}
 	}
 
+	[ExportContextMenuEntry(Header = nameof(Resources._JumpToEntryPoint))]
+	sealed class JumpToEntryPoint : IContextMenuEntry
+	{
+		public bool IsVisible(TextViewContext context)
+		{
+			return GetEntryPoint() != null;
+		}
+
+		public bool IsEnabled(TextViewContext context) 
+		{
+			return true;
+		}
+
+		public void Execute(TextViewContext context)
+		{
+			MethodDefinition entryPoint = GetEntryPoint();
```

Comment:
Here the `context ` parameter is missing as well...
It seems like you did not try to run this code before creating a pull request...